packages/react-native/src/theme/utils.ts

```diff
@@ -0,0 +1,26 @@
+import mapValues from 'lodash/mapValues';
+import get from 'lodash/get';
+import { Tokens } from './types';
+
+/**
+ * Util similar to lodash mapValues but recurses into nested objects.
+ * @param obj
+ * @param callback
+ */
+const mapValuesDeep = <T>(obj: T, callback: Function): T =>
+  typeof obj === 'object'
+    ? (mapValues(obj, (obj) => mapValuesDeep(obj, callback)) as T) // cast back to T to appease TS
```

Comment:
I see `obj` in three places in this line, is that expected?

Reply:
I could rename the second and third, any suggestions?